client/components/tinymce/plugins/embed/dialog.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Dialog from 'components/dialog';
+import FormTextInput from 'components/forms/form-text-input';
+import { localize } from 'i18n-calypso';
```

Comment:
Nit: We usually pile 'i18n-calypso' in to externals since it's source is outside of Calypso (even though really it's calypso specific)